webui/react/src/components/ConfigPolicies.tsx

```diff
@@ -44,55 +46,104 @@ const ConfigPoliciesValues: Record<ConfigPoliciesType, ConfigPoliciesValues> = {
-}
+};
 
 type FormInputs = {
-  configPolicies: string;
+  [YAML_FORM_ITEM_NAME]: string;
 };
 
-const ConfigPolicies: React.FC<Props> = ({ workspaceId }: Props) => {
-  const tabItems: PivotProps['items'] = [
-    {
-      children: <ConfigPoliciesTab type="experiments" workspaceId={workspaceId} />,
-      key: 'experiments',
-      label: ConfigPoliciesValues.experiments.label,
-    },
-    {
-      children: <ConfigPoliciesTab type="tasks" workspaceId={workspaceId} />,
-      key: 'tasks',
-      label: ConfigPoliciesValues.tasks.label,
-    },
-  ];
+const SUCCESS_MESSAGE = 'Config policies updated.';
+const YAML_FORM_ITEM_NAME = 'configPolicies';
+
+const ConfigPolicies: React.FC<Props> = ({ workspaceId, global }: Props) => {
+  let tabItems: PivotProps['items'];
+  if (global) {
+    tabItems = [
+      {
+        children: <ConfigPoliciesTab global type="experiments" />,
+        key: 'experiments',
+        label: ConfigPoliciesValues.experiments.label,
+      },
+      {
+        children: <ConfigPoliciesTab global type="tasks" />,
+        key: 'tasks',
+        label: ConfigPoliciesValues.tasks.label,
+      },
+    ];
+  } else if (workspaceId) {
+    tabItems = [
+      {
+        children: <ConfigPoliciesTab type="experiments" workspaceId={workspaceId} />,
+        key: 'experiments',
+        label: ConfigPoliciesValues.experiments.label,
+      },
+      {
+        children: <ConfigPoliciesTab type="tasks" workspaceId={workspaceId} />,
+        key: 'tasks',
+        label: ConfigPoliciesValues.tasks.label,
+      },
+    ];
+  }
 
   return <Pivot items={tabItems} type="secondary" />;
 };
 
-const ConfigPoliciesTab: React.FC<TabProps> = ({ workspaceId, type }: TabProps) => {
+const ConfigPoliciesTab: React.FC<TabProps> = ({ workspaceId, global, type }: TabProps) => {
   const confirm = useConfirm();
   const { openToast } = useToast();
-  const { canModifyWorkspaceConfigPolicies, loading: rbacLoading } = usePermissions();
+  const {
+    canModifyWorkspaceConfigPolicies,
+    canModifyGlobalConfigPolicies,
+    loading: rbacLoading,
+  } = usePermissions();
   const [form] = Form.useForm<FormInputs>();
 
   const [disabled, setDisabled] = useState(true);
 
-  const APPLY_MESSAGE = "You're about to apply these config policies to this workspace.";
-  const VIEW_MESSAGE = 'An admin applied these config policies to this workspace.';
-  const CONFIRMATION_MESSAGE = 'Config policies updated';
+  const applyMessage = global
+    ? "You're about to apply these config policies to this cluster."
+    : "You're about to apply these config policies to this workspace.";
+  const viewMessage = global
+    ? 'These global config policies are being applied to this cluster.'
+    : 'These global config policies are being applied to this workspace.';
+  const confirmMessageEnding = global
+    ? 'underlying workspaces, projects, and submitted experiments in this cluster.'
+    : 'underlying projects and their experiments in this workspace.';
```

Comment:
@tara-det-ai

Reply:
I would suggest avoiding any abbreviated forms of "configuration" because some users will use a translation service. I would suggest avoiding use of "these/this" in place of "the" because it's awkward and might be confusing for users. Use "the" instead. e.g., "the workspace" or "the cluster" rather than "this cluster" or "this workspace" because it's a more natural way of speaking.

You're about to apply these configuration policies to the cluster.
You're about to apply these configuration policies to the workspace.
Global configuration policies are being applied to the cluster.